cmd/detect.go

```diff
@@ -110,6 +110,16 @@ func runDetect(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	// Remove findings from the baseline (an existing report in json format generated earlier)
+	baselinePath, _ := cmd.Flags().GetString("baseline-path")
+	if baselinePath != "" {
+		findings, err = report.Baseline(findings, baselinePath)
+		if err != nil {
+			log.Info().Msgf("Ignoring --baseline-path flag, because the baseline could not be loaded: %s", err)
```

Comment:
If someone passes a bad baseline path, I think gitleaks should exit immediately with this error. We have to assume they intended to pass a good path and they'd want a noisy failure if the path was bad.

Reply:
Making it fatal with exit code 1.